packages/theme/src/cli/utilities/themes-api.ts

```diff
@@ -24,6 +24,11 @@ export async function updateTheme(id: number, params: ThemeParams, session: Admi
   return buildTheme(response.json.theme)
 }
 
+export async function publishTheme(id: number, session: AdminSession): Promise<Theme | undefined> {
```

Comment:
I think I'd rather leave it explicit. My idea here is that, much like the `renderConfirmation` helper, you wouldn't know otherwise how to publish a theme. You'd have to look up the docs, then make the `updateTheme` call with the right params (which are not really obvious).

That said, if the API call is indeed the same, we could make `publishTheme` call `updateTheme` internally. That way it's a bit more obvious that they are the same without sacrificing the discoverability by intention.

Reply:
Thanks for the clarification, @charlespwd.

+1 for having `publishTheme`  calling  `updateTheme`, or even to remove `updateTheme` in favor of the `publishTheme` function :)